<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualUITestingPage</class>
 <widget class="QWidget" name="ManualUITestingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Testing &amp; Validation</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructionLabel">
     <property name="text">
      <string>&lt;b&gt;Your action is required to complete the UI testing:&lt;/b&gt;
&lt;ol&gt;
&lt;li&gt;Go to the &lt;b&gt;Documents&lt;/b&gt; page to download the &lt;i&gt;UI Test Plan&lt;/i&gt;.&lt;/li&gt;
&lt;li&gt;Execute the tests as described in the document and record the results.&lt;/li&gt;
&lt;li&gt;Upload the completed document below to trigger the final evaluation.&lt;/li&gt;
&lt;/ol&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="goToDocumentsButton">
     <property name="text">
      <string>Go to Documents Page</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="uploadLabel">
     <property name="text">
      <string>&lt;b&gt;Upload Completed Test Plan Results:&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="filePathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="processResultsButton">
     <property name="text">
      <string>Process Test Results</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>